src/components/20-molecules/datepicker/index.js

```diff
@@ -193,6 +191,10 @@ class AXADatepicker extends NoShadowDOM {
     };
   }
 
+  get isControlled() {
```

Comment:
any particular reason for moving this? just curious...

Reply:
yeah also rule that the getter and setter should be next to each other. `get value and set value`